surfactant/relationships/dotnet_relationship.py

```diff
@@ -86,13 +88,29 @@ def establish_relationships(
             if "Name" not in asmRef:
                 continue
             refName = asmRef["Name"]
+
+            # Check absolute path against entries in software
             if is_absolute_path(refName):
-                relationships.extend(get_windows_pe_dependencies(sbom, software, [refName]))
+                ref_abspath = pathlib.PureWindowsPath(refName)
+                for e in sbom.software:
+                    if e.installPath is None:
+                        continue
+                    if isinstance(e.installPath, Iterable):
+                        for ifile in e.installPath:
+                            if ref_abspath == pathlib.PureWindowsPath(ifile):
+                                relationships.extend(
+                                    Relationship(dependent_uuid, e.uuid, "Uses")
+                                )
+                relationships.extend(
+                    get_windows_pe_dependencies(sbom, software, [refName])
+                )
```

Comment:
I don't think this line is needed -- haven't confirmed, but in that function it is appending refName (which in this case is an absolute path such as `C:/Somefolder/Dep.dll`) with the path software is located in this function will just be searching for garbage paths like `C:/PrgFiles/C:/Somefolder/Dep.dll` that are actually invalid path names on Windows (`:` is not allowed in folder names) and can't ever exist.